src/views/new.ecr

```diff
@@ -11,6 +11,13 @@
 
       async function encryptAndStore() {
         const { key, ciphertext } = await encryptLocally(plaintext());
+        const isInputTooLong = to_base64(ciphertext).length > <%= max_ciphertext_length %>;
```

Comment:
My long-term design goal is to move away from ecr-templates but using HTML only. You're going into another direction. Please move this test into the backend. Security relevance is low and you can steer it globally then. Also you can simply show the backend error message for all messages generically.